### ISSUE TITLE
VAEDecodeTiled.decode() missing 1 required positional argument: 'overlap'

### ISSUE THREAD

**Issue by SparklyBird** [ltdrdata/ComfyUI-Impact-Pack#812]
A few days ago was using Iterative upscale for Flux images, but now I am getting an error on node "Iterative Upscale (Image)" saying that it missing argument "overlap". Here is a error report:

# ComfyUI Error Report
## Error Details
- **Node Type:** IterativeImageUpscale
- **Exception Type:** TypeError
- **Exception Message:** VAEDecodeTiled.decode() missing 1 required positional argument: 'overlap'
## Stack Trace
```
  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)

  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\impact_pack.py", line 1315, in doit
    refined_latent = IterativeLatentUpscale().doit(latent, upscale_factor, steps, temp_prefix, upscaler, step_mode, unique_id)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\impact_pack.py", line 1267, in doit
    current_latent = upscaler.upscale_shape(step_info, current_latent, new_w, new_h, temp_prefix)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 1708, in upscale_shape
    latent_upscale_on_pixel_space_with_model_shape2(samples, scale_method, self.upscale_model,

  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 1426, in latent_upscale_on_pixel_space_with_model_shape2
    pixels = vae_decode(vae, samples, use_tile, hook, tile_size=tile_size)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 1359, in vae_decode
    pixels = nodes.VAEDecodeTiled().decode(vae, samples, tile_size)[0]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```
## System Information
- **ComfyUI Version:** v0.2.7-6-g2865f91
- **Arguments:** main.py --use-pytorch-cross-attention --bf16-vae --listen --port 8188 --preview-method auto
- **OS:** nt
- **Python Version:** 3.12.4 (tags/v3.12.4:8e8a4ba, Jun  6 2024, 19:30:16) [MSC v.1940 64 bit (AMD64)]
- **Embedded Python:** false
- **PyTorch Version:** 2.5.0.dev20240815+cu118
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 3080 : cudaMallocAsync
  - **Type:** cuda
  - **VRAM Total:** 10736893952
  - **VRAM Free:** 5404300986
  - **Torch VRAM Total:** 4026531840
  - **Torch VRAM Free:** 11474618

## Logs
```
2024-11-07 23:23:14,436 - root - ERROR - !!! Exception during processing !!! VAEDecodeTiled.decode() missing 1 required positional argument: 'overlap'
2024-11-07 23:23:14,466 - root - ERROR - Traceback (most recent call last):
  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "A:\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\impact_pack.py", line 1315, in doit
    refined_latent = IterativeLatentUpscale().doit(latent, upscale_factor, steps, temp_prefix, upscaler, step_mode, unique_id)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\impact_pack.py", line 1267, in doit
    current_latent = upscaler.upscale_shape(step_info, current_latent, new_w, new_h, temp_prefix)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 1708, in upscale_shape
    latent_upscale_on_pixel_space_with_model_shape2(samples, scale_method, self.upscale_model,
  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 1426, in latent_upscale_on_pixel_space_with_model_shape2
    pixels = vae_decode(vae, samples, use_tile, hook, tile_size=tile_size)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "A:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 1359, in vae_decode
    pixels = nodes.VAEDecodeTiled().decode(vae, samples, tile_size)[0]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: VAEDecodeTiled.decode() missing 1 required positional argument: 'overlap'

**Comment by SparklyBird** [comment]
Workflow:

![384146574-bc0d9df2-678f-4349-a7e8-8e15abea6ed1](https://github.com/user-attachments/assets/87401a3c-90dd-4a25-a492-2d29f06a9364)

![384146559-1373586b-300b-4811-a6fe-1a876e93c226](https://github.com/user-attachments/assets/870ba6e0-8ba0-477d-902a-541ceb177989)

![384146565-055966a6-4730-4a94-a3cc-fe188368f86f](https://github.com/user-attachments/assets/f8943a2e-8d4f-4698-99c2-c34b532c94f0)

**Comment by ltdrdata** [comment]
This is patched on v7.11.1.
